adafruit_rgb_display/rgb.py

```diff
@@ -80,7 +80,7 @@ def fill_rectangle(self, x, y, width, height, color):
         w = min(self.width - x, max(1, width))
         h = min(self.height - y, max(1, height))
         self._block(x, y, x + w - 1, y + h - 1, b'')
-        chunks, rest = divmod(w * h, 512)
+        chunks, rest = divmod(w * h, 256)
```

Comment:
Could you add a comment explaining the buffer choice size?

Reply:
I moved it into a constant and added explanation.